### DISCUSSION TITLE
posts/kmeans-clustering

### DISCUSSION THREAD

**Discussion by giscus** [acciochris/acciochris.github.io#9] [Announcements]
# posts/kmeans-clustering

https://acciochris.github.io/posts/kmeans-clustering.html

**Comment by acciochris** [comment]
Please let me know if there are any errors in this post :grinning:. I'm not sure whether I did the data analysis part right. (I had to manually inspect hundreds of diagrams :sleepy:)

**Comment by picopeak** [comment]
Why are there 3 steps for well-clustered and poor-clustered examples? Or you just manually run 3 iterations for each case?

**Reply by acciochris** [reply]
The kmeans algorithm itself is an iterative algorithm (see implementation above). I named the three iterations (before break) as steps.
